src/js/popup.js

```diff
@@ -339,6 +342,98 @@ $(document).ready(function () {
         });
     });
 
+    $('#customDomains').click(function () {
+        dmUIClick('customDomains');
+        if (!$(this).hasClass('disabled')) {
+            $('.customDomainList').toggle();
+            $('main').toggle();
+        } else {
+            Materialize.toast(disabledOptionToast, 2000, 'toastNotification');
+        }
+    });
+
+    $('#customDomainsBackButton').click(function () {
+        dmUIClick('customDomainsBackButton');
+        $('.customDomainList').toggle();
+        $('main').toggle();
+    });
+
+    $('#clearCustomDomains').click(function () {
+        dmUIClick('clearCustomDomains');
+        // remove all added domains:
+        chrome.runtime.sendMessage({
+            JDTIfunction: 'removeDomain',
+            domainName: '',
```

Comment:
Just throwing it out there. If we validate that no empty string can be added as a domain, then we could use the empty string in the domainName to indicate removeAll

Reply:
While this is correct, I would argue that it is clearer for future developers to have an obvious parameter which triggers removeAll or not. Also, I prefer having the logic separate from the data, so that if future changes to the data occur the logic remains intact. Feel free to contradict me though.

Reply:
That's fine with me.